### PR TITLE
fix: document cant be selected

### PR DESCRIPTION
**Pull request by Ziprion**
Closes [#729](https://github.com/epam/badgerdoc/issues/729)

### CODE REVIEW

File: web/src/components/documents/document-card-view-item/document-card-view-item.module.scss [hunk 1]
```diff
@@ -2,7 +2,7 @@
     position: relative;
```
Comment (iSokrat) [inline web/src/components/documents/document-card-view-item/document-card-view-item.module.scss:2]: Do we really need to show this border by default? Should id be only appear when we drop something there?
image

Comment (iSokrat) [inline web/src/components/documents/document-card-view-item/document-card-view-item.module.scss:2]: Just noticed that the color of ticks was changed to grey (not sure that this PR affected this but maybe some previous)?
image

On dev2 we have white ticks
image

File: web/src/pages/documents/documents-page.tsx [hunk 2]
```diff
@@ -95,14 +95,15 @@ const DocumentsPage = () => {
...
+                                    flexDirection: 'column'
+                                }}
                             >
-                                {!isFileOver && (
-                                    <div
-                                        style={{
-                                            height: '100%'
-                                        }}
-                                    >
+                                <DocumentsDropZone dataset={activeDataset}>
```
Comment (iSokrat) [inline web/src/pages/documents/documents-page.tsx:105]: Do you know how should drag & drop work when we move file via "in documents" tab? I've tried to move usual jpg there, it was successfully loaded but shown in "in file names" tab. Should we somehow automatically redirect the user to the place where it was placed? Or it should appear in "in documents" tab, do you know? Is drag & drop actually needed for "in documents" tab or not?

Comment (Ziprion) [reply web/src/pages/documents/documents-page.tsx:105]: Actually, I don't know how it should work. Anyway, it's out of scope of this PR

File: web/src/components/documents/document-card-view-item/document-card-view-item.module.scss (hunk 1 above)
Comment (Ziprion) [reply web/src/components/documents/document-card-view-item/document-card-view-item.module.scss:2]: I think, we should show that it's a dnd zone and user could use this feature. In additional, we have different styles for dnd zone during overing files(blue borders and background)

Comment (iSokrat) [reply web/src/components/documents/document-card-view-item/document-card-view-item.module.scss:2]: Okay,  let's keep it

File: web/src/pages/documents/documents-page.tsx (hunk 2 above)
Comment (iSokrat) [reply web/src/pages/documents/documents-page.tsx:105]: Okay, moved this to "Open questions"

File: web/src/components/documents/document-card-view-item/document-card-view-item.module.scss (hunk 1 above)
Comment (Ziprion) [reply web/src/components/documents/document-card-view-item/document-card-view-item.module.scss:2]: fixed

**Review by iSokrat — APPROVED** [review]

**Pull request merged**